Add Camino Messenger Account and Partner configuration to glossary (#317)
chain4travel/camino-docs: docs/faq/glossary.md

The backbone of the Camino Messenger infrastructure. Essentially, it's a server
relaying messages between bots. Currently, Matrix messenger server is used.

## Camino Messenger Account

The Camino Messenger Account is a smart contract (actually an ensemble of smart contracts), acting as the financial backbone of the [Partner Configuration](#partner-configuration), holding the funds designated for trading on behalf of the Partner on the Camino Messenger, and paying / receiving payment amounts for bookings.  
It also stores all configuration settings such as the services offered and wanted (and their versions), the applicable service fees and the accepted currencies. It also maintains a list of the wallets of the off-chain bots with which the Partner's systems interact with the Messenger.

The Camino Messenger Account can be created from the Partner Showroom of the [Camino Suite](https://suite.camino.network/partners) or - for development purposes on Columbus only - by using a [cli tool](https://github.com/chain4travel/camino-messenger-bot?tab=readme-ov-file#camino-messenger-account-cm-account)

> More details can be found in our functional documentation: [Camino Partner Showroom](https://docs.camino.network/partners/partner-showroom "Camino Partner Showroom")

Author: Does this affect SEO? Is it ok having both links with index.html and links without index.html available on the page? We already have the links without index.html on the site, this change would introduce links with index.html.

> More details can be found in our functional documentation: [Camino Partner Showroom](https://docs.camino.network/partners/partner-showroom "Camino Partner Showroom")

Author: @coderabbitai Does this affect SEO? Is it ok having both links with index.html and links without index.html available on the page? We already have the links without index.html on the site, this change would introduce links with index.html.

The backbone of the Camino Messenger infrastructure. Essentially, it's a server
relaying messages between bots. Currently, Matrix messenger server is used.

## Camino Messenger Account

The Camino Messenger Account is a smart contract (actually an ensemble of smart contracts), acting as the financial backbone of the [Partner Configuration](#partner-configuration), holding the funds designated for trading on behalf of the Partner on the Camino Messenger, and paying / receiving payment amounts for bookings.  
It also stores all configuration settings such as the services offered and wanted (and their versions), the applicable service fees and the accepted currencies. It also maintains a list of the wallets of the off-chain bots with which the Partner's systems interact with the Messenger.

> 

Reviewer: The Camino Messenger Account can be created from the Partner Showroom of the [Camino Suite](https://suite.camino.network/partners) or - for development purposes on Columbus only - by using a [script](https://github.com/chain4travel/camino-messenger-bot?tab=readme-ov-file#camino-messenger-account-cm-account)

More details can be found in our functional documentation: [Camino Partner Showroom](https://docs.camino.network/partners/partner-showroom "Camino Partner Showroom")

Author: @SamJaarsma 
@havan 

I would avoid mentioning the script if it is only meant/encouraged to be used for development purposes (test accounts). Or we mention that here. 

If it's ok that mainnet account are created via hardhat, we can keep it but switch the link to c4t branch once the dev is merged.

Reviewer: Agreed that live accounts should not be used for Camino, only for development on Columbus. Updated the suggestion to include that

Author: done

Author: done

> ## Camino Messenger Protocol

Reviewer: move "Camino Messenger Protocol" to before "Partner Plugin"

Author: I put acronym CMB before CMP

Author: @SamJaarsma I'm not sure what you mean
I see that
Camino Messenger Protocol line `89`
Partner Plugin line `448`

But I did notice that "Camino Messenger Account" was below "Camino Messenger Bot", so I changed that.

> research plan and book various travel-related services and products often offering a

Reviewer: ```suggestion
planning and booking various travel-related services and products often offering a
```

> wide range of options and choices from different suppliers.

Reviewer: ```suggestion
wide range of options and choices from different suppliers.

More details can be found in our functional documentation: [Camino Messenger Protocol Documentation](https://docs.camino.network/camino-messenger/introduction)
```

## Partner Configuration

The Partner Configuration allows Partners to make themselves known to the Camino Network as service providers, or as looking for services from other Partners in the Network. Upon matching between services wanted and offered by two parties, such services will be then traded on Camino Messenger.

> 

Reviewer: ```suggestion
More details can be found in our functional documentation: [Camino Partner Configuration](https://docs.camino.network/partners/partner-config "Camino Partner Configuration")
```

## Partner plugin

> The Partner Plugin serves as a bridge enabling the Camino Messenger Bot to integrate

Reviewer: ```suggestion
The Partner Plugin serves as a bridge enabling the Camino Messenger Bot to integrate with the Partner distribution or inventory system.

More details can be found in our functional documentation: [Camino Messenger Bot Partner Plugin](https://docs.camino.network/camino-messenger/bot/partner-plugin/ "Camino Messenger Bot Partner Plugin")
```

Author: @SamJaarsma this link is invalid:
https://docs.camino.network/partners/partner-plugin

Did you mean this one: https://docs.camino.network/camino-messenger/bot/partner-plugin/ ?

Reviewer: oops, correct!

Author: done

## OTA

Online Travel Agency. OTAs are online platforms or websites that allow consumers to
planning and booking various travel-related services and products often offering a
wide range of options and choices from different suppliers.

> More details can be found in our functional documentation: [Camino Messenger Protocol Documentation](https://docs.camino.network/camino-messenger/introduction)

Reviewer 2: @VjeraTurk @SamJaarsma this looks valid to me, why we have it under OTA section?

> More details can be found in our functional documentation: [Camino Messenger Protocol Documentation](https://docs.camino.network/camino-messenger/introduction)

Reviewer: Yes I agree, line 448 should be included below line 97

> More details can be found in our functional documentation: [Camino Messenger Protocol Documentation](https://docs.camino.network/camino-messenger/introduction)

Author: done

## OTA

Online Travel Agency. OTAs are online platforms or websites that allow consumers to
planning and booking various travel-related services and products often offering a

> wide range of options and choices from different suppliers.

Author: @coderabbitai Check again
